configs.go

```diff
@@ -1277,3 +1277,23 @@ func (config GetStickerSetConfig) values() (url.Values, error) {
 	v.Add("name", config.Name)
 	return v, nil
 }
+
+// DiceConfig contains information about a sendDice request.
+// Chattable interface
+type DiceConfig struct {
+	BaseChat
+}
+
+// values returns a url.Values representation of ForwardConfig.
```

Comment:
This comment needs to be fixed.

```suggestion
// values returns a url.Values representation of DiceConfig.
```